src/AudioPlayer.cpp

```diff
@@ -1067,21 +1057,15 @@ size_t AudioPlayer_NvsRfidWriteWrapper(const char *_rfidCardId, const char *_tra
 
 // Adds webstream to playlist; same like SdCard_ReturnPlaylist() but always only one entry
 char **AudioPlayer_ReturnPlaylistFromWebstream(const char *_webUrl) {
-	char *webUrl = x_strdup(_webUrl);
-	static char **url;
-
-	if (url != NULL) {
-		--url;
-		freeMultiCharArray(url, strtoul(*url, NULL, 10));
-	}
+	static char number[] = "1";
+	static char *url[2] = {number, nullptr};
 
-	url = (char **)x_malloc(sizeof(char *) * 2);
+	free(url[1]);
 
-	url[0] = x_strdup("1"); // Number of files is always 1 in url-mode
-	url[1] = x_strdup(webUrl);
+	number[0] = '1';
+	url[1] = x_strdup(_webUrl);
 
-	free(webUrl);
-	return ++url;
+	return &(url[1]);
```

Comment:
Ok, I got it now. This looks ok to me. In the long run we need to get rid of those static variables.